package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-skeleton-content",
-  "version": "1.0.24",
+  "version": "1.1.0",
```

Comment:
Not sure how you do versioning. Let me know if I should change it to be just "1.0.25"

Reply:
Will probably have to `npm publish` after that, changing the version automatically, we can keep this as it is and will create a follow-up commit to it!